src/test/java/com/puppycrawl/tools/checkstyle/internal/AllChecksTest.java

```diff
@@ -405,6 +405,7 @@ private static void validateAllCheckTokensAreReferencedInConfigFile(String confi
 
         final Map<String, Set<String>> configCheckTokens = new HashMap<>();
         final Map<String, Set<String>> checkTokens = new HashMap<>();
+        AbstractCheck check;
```

Comment:
Please move this variable closer to its use. And make it final.

Reply:
Done